tests/compas/geometry/test_frame.py

```diff
@@ -1,14 +1,17 @@
 from __future__ import division
+
+import math
+
 import pytest
 import json
 import compas
 from random import random
-from compas.geometry import allclose
+from compas.geometry import allclose, Rotation
```

Comment:
We have this `isort` setting that forces all imports to be single-line. The setting is at the top of the repo, so, if you have installed isort, it should auto-sort them. In VSCode, you can do `right-click -> Organizer imports`:

```suggestion
from compas.geometry import allclose
from compas.geometry import Rotation
```

Reply:
@gonzalocasas thx for the VSCode heads up, alas, no pycharm equivalent.
what about embracing `pre-commit` hooks?
[seems a super clean way of enforcing code standards?](https://gist.github.com/guptarohit/35f49bc663620d03ca00d2a0ff7429df)

Reply:
Totally in for `pre-commit` hooks, I just didn't have the time to set that up yet